src/redisearch_api.c

```diff
@@ -161,6 +161,21 @@ QueryNode* RS_CreatePrefixNode(IndexSpec* sp, const char* fieldName, const char*
   return ret;
 }
 
+QueryNode* RS_CreateLexRangeNode(IndexSpec* sp, const char* fieldName, const char* begin,
+                                 const char* end) {
```

Comment:
you need to register this function, look how I registered all the other functions.